default.nix

```diff
@@ -85,6 +85,77 @@ let
       });
     };
   });
+  rawDockerImage = let
+    topologyFile = pkgs.writeText "topology.yaml" ''
+      wallet:
+        relays:
+          [
+            [
+              { host: nodes.awstest.iohkdev.io }
+            ]
+          ]
+        valency: 1
+        fallbacks: 7
+    '';
+    CLUSTER = "mainnet-1.0";
```

Comment:
The issue says to target staging.

Reply:
@cleverca22 don't think we really need to suffix state dirs in docker